app/src/org/commcare/utils/FormUploadUtil.java

```diff
@@ -390,6 +394,9 @@ private static String getFileContentType(File f) {
         return MultipartBody.Part.createFormData(partName, file.getName(), requestFile);
     }
 
+    public static boolean isRunningTest() {
+        return CommCareApplication.instance().isRunningTest();
```

Comment:
You can just check if `CommCareApplication` instance of `CommCareInstrumentationTestApplication`

Reply:
😅yeah. Thanks for adding that.